test/WebSites/ModelBindingWebSite/Startup.cs

```diff
@@ -17,6 +18,10 @@ public void Configure(IApplicationBuilder app)
             // Set up application services
             app.UseServices(services =>
             {
+                // Override the IModelMetadataProvider AddMvc would normally use. Make service a singleton though
+                // AddMvc would configure DataAnnotationsModelMetadataProvider as transient.
```

Comment:
Changing DAMMP to a singleton is currently out for PR. IMO just call out the tests that depend on this custom provider, the rest of this comment doesn't say much about WHY this matters.